### PR TITLE
Add an example of fetching schema.json from server

### PR DESCRIPTION
**Pull request by Vanuan**
It's great to have an example of how you download schema.json from graphql server.

### CODE REVIEW

**Comment by facebook-github-bot-9** [comment]
Thank you for your pull request and welcome to our community. We require contributors to sign our Contributor License Agreement, and we don't seem to have you on file. In order for us to review and merge your code, please sign up at https://code.facebook.com/cla - and if you have received this in error or have any questions, please drop us a line at cla@fb.com. Thanks!

**Comment by Vanuan** [comment]
signed

**Comment by facebook-github-bot-9** [comment]
Thank you for signing our Contributor License Agreement. We can now accept your code for this (and any) Facebook open source project. Thanks!

**Comment by Vanuan** [comment]
An alternative way:

`wget --header='content-type:application/json' --header='accept:application/json' --post-data='{"query":"\n  query IntrospectionQuery {\n    __schema {\n      queryType { name }\n      mutationType { name }\n      subscriptionType { name }\n      types {\n        ...FullType\n      }\n      directives {\n        name\n        description\n        locations\n        args {\n          ...InputValue\n        }\n      }\n    }\n  }\n\n  fragment FullType on __Type {\n    kind\n    name\n    description\n    fields(includeDeprecated: true) {\n      name\n      description\n      args {\n        ...InputValue\n      }\n      type {\n        ...TypeRef\n      }\n      isDeprecated\n      deprecationReason\n    }\n    inputFields {\n      ...InputValue\n    }\n    interfaces {\n      ...TypeRef\n    }\n    enumValues(includeDeprecated: true) {\n      name\n      description\n      isDeprecated\n      deprecationReason\n    }\n    possibleTypes {\n      ...TypeRef\n    }\n  }\n\n  fragment InputValue on __InputValue {\n    name\n    description\n    type { ...TypeRef }\n    defaultValue\n  }\n\n  fragment TypeRef on __Type {\n    kind\n    name\n    ofType {\n      kind\n      name\n      ofType {\n        kind\n        name\n        ofType {\n          kind\n          name\n        }\n      }\n    }\n  }\n"}' http://$SERVER/graphql/? -O schema.json
`

File: docs/Guides-BabelPlugin.md [hunk 1]
```diff
@@ -109,6 +109,9 @@ For a complete example of how to load a `schema.js` file, run the introspection
...
+    .then(res => saveJSON(res.json()))
+
```
Comment (josephsavona) [inline docs/Guides-BabelPlugin.md]: the introspection query is [exported by graphql-js](https://github.com/graphql/graphql-js/blob/master/src/index.js#L151), so it's probably better to use that. something like 

```
const {introspectionQuery} = require('graphql');
```

**Comment by wincent** [comment]
Closing this one as it got taken care of in https://github.com/facebook/relay/commit/5729cc69266d6c18c.

Thanks!

**Pull request closed without merging**